curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/6464aabd17cd45f1d17cfe56.md

```diff
@@ -10,7 +10,7 @@ dashedName: step-101
 For the object in the middle of the stack, add the property `msg` set to the following string:
 
 ```md
-decimalToBinary(2) returns "1" + 0 (2 % 2) and gives that value to the stack below. Then it pops off the stack.
+`decimalToBinary(2) returns "1" + 0 (2 % 2) and gives that value to the stack below. Then it pops off the stack.`
```

Comment:
```suggestion
`"decimalToBinary(2) returns '1' + 0 (2 % 2) and gives that value to the stack below. Then it pops off the stack."`
```

Reply:
There are triple backticks already, I've removed the single backticks around double quotes